lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/EvaluatorSettings.cs

```diff
@@ -174,12 +187,46 @@ public IRemoteManager<REEFMessage> RemoteManager
             }
         }
 
-        public IInjector Injector
+        public IInjector ClrDriverConfigInjector
         {
             get
             {
                 return _injector;
             }
         }
+
+        private Optional<TaskConfiguration> GetTaskConfiguration()
```

Comment:
Usually either service Configuration is given or task configuration is given when evaluate started. As both are string, overloading constructor cannot distinguish those two cases.